packages/app/components/Nav/NavBar.tsx

```diff
@@ -119,16 +125,55 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
     setVisible(false);
   };
 
+  const courseSelector = (
+    <Menu>
+      {profile?.courses.map((c) => (
+        <Menu.Item key={c.course.id}>
+          <Link href="/course/[cid]/today" as={`/course/${c.course.id}/today`}>
```

Comment:
anyone know a guy who can do dropdowns? @zefwang

Reply:
I think you're looking for @amielmon. Alternatively, copy the code :)